src/plugins/intel_cpu/src/nodes/executors/fullyconnected_implementations.cpp

```diff
@@ -305,6 +320,37 @@ const std::vector<ExecutorImplementation<FCAttrs>>& getImplementations() {
                     context,
                     false);
             })
+        // Temporary disable FC ACL ARM64 executor until add repack weights (add has_opt_impl())
```

Comment:
Seems like the comment is not relevant anymore?

Reply:
@dmitry-gorokhov remvoed